geode-core/src/main/java/org/apache/geode/cache/query/internal/QRegion.java

```diff
@@ -321,7 +321,7 @@ public void destroyRegion(Object aCallbackArgument)
   }
 
   public Set entries(boolean recursive) {
```

Comment:
Another example of a method that should no longer exist since it is no longer on Region